plugin/storage/es/factory.go

```diff
@@ -230,15 +234,20 @@ func (f *Factory) CreateArchiveSpanWriter() (spanstore.Writer, error) {
 	if !f.archiveConfig.Enabled {
 		return nil, nil
 	}
-	return createSpanWriter(f.getArchiveClient, f.archiveConfig, true, f.archiveMetricsFactory, f.logger)
+	writeAlias := "archive"
+	if f.archiveConfig.UseReadWriteAliases {
+		writeAlias += "-write"
+	}
+	return createSpanWriter(f.getArchiveClient, f.archiveConfig, f.archiveMetricsFactory, f.logger, writeAlias, true)
```

Comment:
@yurishkuro With the latest change, we may actually not even need to hardcode `use_aliases` to true anymore since the only differentiator that remains is the actual alias that is used. If we want, we can change the ES reader to just take an optional suffix and then do something like this in `getTimeRangeIndexFn`

```
if (suffix) {
  if (use_aliases) {
    return suffix + read
  }
return suffix 
}
return time ranged indices
```

Do you have a preference between the approach we currently have or the one that I just described?